examples/node-groups/managed-node-groups/main.tf

```diff
@@ -126,7 +126,7 @@ module "eks_blueprints" {
       public_ip         = false # Use this to enable public IP for EC2 instances; only for public subnets used in launch templates
 
       http_endpoint               = "enabled"
-      http_tokens                 = "optional"
+      http_tokens                 = "required"
```

Comment:
why this change?

Reply:
to introduce IMDSv2 which is a security best practice

Reply:
Or we can have it as a variable please instead of being hard coded

Reply:
in the next [iteration of blueprints](https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/1319), we will be utilizing the `terraform-aws-eks` module for cluster and node group creation and this module already has the default values set that you are looking for https://github.com/terraform-aws-modules/terraform-aws-eks/blob/a8e0de0b3f3a3e27232b35e9ed8814549273bced/node_groups.tf#L2-L6